Add product logos to logo library

audroue opened the pull request.

audroue committed:
Added product logos to logo library

- Renamed logo file to logos
- Added cloud manager, atlas, charts, and realm product icons

audroue committed:
Renamed logo file to logos

DesignerDave reviewed (changes requested):
Hey Rob!

These look pretty good. There's two notes I had:

- Not all of the logos have a layer style override for their background color. Additionally, for the ones that do, the layer you can override the layer style of has names like "Combined Shape Copy"
- An idea that you can take or leave: instead of nesting the product logos so deeply (`Logos [LeafyGreen] / Product Logos / Atlas / Color`), you could put them at the same level as the leaf and such, so it would be `Logos [LeafyGreen] / Atlas / Color`.

audroue committed:
Updated library based on feedback

- Ensured all gray icons allow for color override and changed name of override to Logo Color
- Added reference symbol
- Simplified nesting of symbols for better findability

audroue committed:
Reordered symbol nesting

Reordered symbol nesting

DesignerDave reviewed (approved):
Thanks Rob!
